drivers/bmp180/bmp180_saul.c

```diff
@@ -42,9 +39,7 @@ static int read_pressure(void *dev, phydat_t *res)
 {
     bmp180_t *d = (bmp180_t *)dev;
 
-    bmp180_read_pressure(d, &pressure);
-
-    res->val[0] = pressure/10;
+    res->val[0] = bmp180_read_pressure(d) / 10;
```

Comment:
This is a hack because `res->val` contains int16 values and pressure returns uint32. Pressure is return in Pa, e.g around 100000. Dividing by 10 make them fit in int16.
But, of course, the unit below is wrong, it should be daPa (decapascal).